nginx/nginx.conf

```diff
@@ -20,4 +20,13 @@ server {
     location /api {
 	    proxy_pass http://backend:8888/api;
 	}
+
+	location /health {
+	    proxy_pass http://backend:8888/health;
+	}
+
+	location ~ ^/test/(.*)$ {
+	    resolver 8.8.8.8;
+	    proxy_pass https://httpbin.org/$1$is_args$args;
+	}
```

Comment:
This was to test to test nginx on its own. I was wondering why things weren't working, but looked like it must. So I tried this locally:

![image](https://user-images.githubusercontent.com/96808/167211715-6484b370-4bcc-4417-9918-2c8806826ec5.png)


Then I tried it on the EB public URL, and it didn't work. Then after I fixed the health check, the above, and frontend+API endpoints started working. Ruddy AWS!